packages/react-router/src/ReactRouter/IonRouter.tsx

```diff
@@ -173,13 +172,15 @@ class IonRouterInner extends React.PureComponent<IonRouteProps, IonRouteState> {
           const currentRouteInfo = this.locationHistory.current();
           routeInfo.lastPathname = currentRouteInfo?.pathname || routeInfo.lastPathname;
           routeInfo.pushedByRoute = currentRouteInfo?.pushedByRoute || routeInfo.pushedByRoute;
-          routeInfo.routeDirection = currentRouteInfo?.routeDirection || routeInfo.routeDirection;
-          routeInfo.routeAnimation = currentRouteInfo?.routeAnimation || routeInfo.routeAnimation;
```

Comment:
Do custom `routeAnimation` props still work here?

Reply:
ya, potentially. This is what was causing the error though. I asked a question on the bug issue around the use case for doing navigations with replaces. I might put this back in based on the response to my question.